Resources/Prototypes/Voice/auto_emotes.yml

```
@@ -16,4 +19,12 @@
   emote: Scream
   interval: 5.0
   chance: 0.5
-  withChat: false
\ No newline at end of file
+  withChat: false
+
+# Brullbar # TODO put this into goob directory
+- type: autoEmote
+  id: BrullbarHowl
+  emote: Scream
+  interval: 5.0 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)